remotemonitoring/armtemplates/basic-static-map.json

```diff
@@ -411,6 +411,19 @@
                             "priority": 100,
                             "direction": "Inbound"
                         }
+                    },
+                    {
+                        "name": "SSH",
+                        "properties": {
+                            "protocol": "TCP",
+                            "sourcePortRange": "*",
+                            "destinationPortRange": "22",
+                            "sourceAddressPrefix": "*",
+                            "destinationAddressPrefix": "*",
+                            "access": "Deny",
+                            "priority": 101,
```

Comment:
what is priority 101?

Reply:
networking rules have a priority from 1 (highest priority) to 4096 (lowest priority).  We're using 100 to open HTTPS, and 101 to open SSH, leaving room before and after for other rules.